docs/governance/templates/release_issue.md

```diff
@@ -46,11 +46,16 @@ and copy it into a release issue. Fill in relevant values, found inside {}
    `make sdk-publish-csharp RELEASE_VERSION={version}-rc`).
    Will need [NuGet API Key](https://www.nuget.org/account/apikeys) from Agones account.
 - [ ] Run `make do-release`. (if release candidate: `make do-release RELEASE_VERSION={version}-rc`) to create and push the docker images and helm chart.
-- [ ] Do a `helm repo add agones https://agones.dev/chart/stable` and verify that the new version is available via the command `helm search agones/`
-- [ ] Do a `helm install` and a smoke test to confirm everything is working.
+- [ ] Do a `helm repo add agones https://agones.dev/chart/stable` / `help repo update` and verify that the new
+ version is available via the command `helm search repo agones --versions --devel`.
+- [ ] Do a `helm install --namespace=agones agones agones/agones` 
+    (`helm install --namespace=agones agones agones/agones --version={version}-rc` if release candidate) and a smoke
```

Comment:
should this be `--namespace=agones-system` as shown in https://agones.dev/site/docs/installation/install-agones/helm/?

Reply:
Yep! Nice catch!